<?xml version='1.0' encoding='UTF-8'?>
<root>
 <tabbed_widget name="Main Window" parent="main_window">
  <plotmatrix rows="2" columns="2" tab_name="plot">
   <plot col="0" row="0">
    <range bottom="-0.065970" left="0.000000" right="7.808000" top="0.004393"/>
    <limitY/>
    <curve name="/uav1/mavros/local_position/odom/twist/twist/linear/x" G="0" R="255" B="0"/>
    <curve name="/uav1/mavros/setpoint_velocity/cmd_vel_unstamped/linear/x" G="128" R="0" B="128"/>
    <transform value="noTransform"/>
   </plot>
   <plot col="0" row="1">
    <range bottom="-0.336478" left="0.000000" right="7.808000" top="1.392600"/>
    <limitY/>
    <curve name="/uav1/mavros/local_position/odom/twist/twist/linear/z" G="0" R="0" B="255"/>
    <curve name="/uav1/mavros/setpoint_velocity/cmd_vel_unstamped/linear/z" G="128" R="128" B="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot col="1" row="0">
    <range bottom="-0.061870" left="0.000000" right="7.808000" top="0.001509"/>
    <limitY/>
    <curve name="/uav1/mavros/local_position/odom/twist/twist/linear/y" G="128" R="0" B="0"/>
    <curve name="/uav1/mavros/setpoint_velocity/cmd_vel_unstamped/linear/y" G="160" R="160" B="164"/>
    <transform value="noTransform"/>
   </plot>
   <plot col="1" row="1">
    <range bottom="-0.023805" left="0.000000" right="7.808000" top="0.022433"/>
    <limitY/>
    <curve name="/uav1/mavros/local_position/odom/twist/twist/angular/z" G="0" R="255" B="255"/>
    <curve name="/uav1/mavros/setpoint_velocity/cmd_vel_unstamped/angular/z" G="0" R="0" B="128"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <plotmatrix rows="2" columns="1" tab_name="plot">
   <plot col="0" row="0">
    <range bottom="0.841051" left="0.012000" right="6.164000" top="1.385664"/>
    <limitY/>
    <curve name="/uav2/errorF/data" G="0" R="128" B="0"/>
    <transform value="noTransform"/>
   </plot>
   <plot col="0" row="1">
    <range bottom="1.308131" left="0.026000" right="6.158000" top="2.016875"/>
    <limitY/>
    <curve name="/uav1/desiredDist/data" G="128" R="0" B="0"/>
    <curve name="/uav1/errorDist/data" G="0" R="0" B="255"/>
    <transform value="noTransform"/>
   </plot>
  </plotmatrix>
  <currentPlotMatrix index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <Plugins>
  <DataLoad_CSV>
   <default time_axis=""/>
  </DataLoad_CSV>
  <DataLoad_ROS_bags>
   <selected_topics list=""/>
  </DataLoad_ROS_bags>
  <ROS_Topic_Streamer>
   <selected_topics list="/uav1/errorDist;/uav1/mavros/local_position/odom;/uav1/mavros/setpoint_velocity/cmd_vel_unstamped;/uav2/errorF;/uav1/desiredDist"/>
  </ROS_Topic_Streamer>
  <RosoutPublisherROS/>
  <TopicPublisherROS/>
 </Plugins>
 <previouslyLoadedStreamer name="ROS_Topic_Streamer"/>
</root>

